src/docs/product/sentry-basics/search/searchable-properties.mdx

```diff
@@ -69,133 +76,145 @@ Release properties overlap with event and issue search properties, so they don't
+| stack.function | Name of the function being called. In events, this is an array. In issues, this is a single value. | x | x |  | array, single value |
+| stack.in_app | Indicates whether a frame is related to the execution of the relevant code in the stack trace. For example, the frames that might power the framework’s web server of your app are probably not relevant. However, calls to the framework’s library once you start handling code likely are relevant. Values can be `1` (true) or `0` (false). | x |  |  | array |
+| stack.lineno | Line number of the call, starting at 1. | x |  |  | array |
+| stack.module | Platform-specific module path. For example, `sentry.interfaces.Stacktrace`. In events, this is an array. In issues, this is a single value. | x | x |  | array, single value |
+| stack.package | The "package" the frame was contained in. Depending on the platform, this can be different things. For C#, it can be the name of the assembly. For native code, it can be the path of the dynamic library or something else. In events, this is an array. In issues, this is a single value. | x | x |  | array, single value |
+| sum(numeric field) | Returns results with a matching total value for the the field entered. | x |  |  | matches field |
+| timesSeen | Returns results with a matching count. (Same as `count()` in events.) |  | x |  | number |
+| timestamp | The finish timestamp of the transaction. Returns events with matching datetime. | x | x |  | datetime |
+| timestamp.to_day | Timestamp rounded down to the nearest day. | x |  |  | datetime |
+| timestamp.to_hour | Timestamp rounded down to the nearest hour. | x |  |  | datetime |
+| title | Title of the error or the transaction name. | x | x |  | string |
+| trace | A trace represents the record of the entire operation you want to measure or track — like page load, searched using the uuid generated by Sentry’s SDK. | x | x | x | uuid |
+| trace.parent_span | Span ID of the parent to the current transaction. This is null if the transaction is root. | x |  |  | uuid |
+| trace.span | Span ID of the root span of the root transaction in the event. | x |  |  | uuid |
+| transaction | For [transactions](/product/performance/transaction-summary/#what-is-a-transaction), the name of the transaction. For errors, the name of the associated transaction. | x | x |  | string |
+| transaction.duration | Duration, in milliseconds, of the transaction. | x |  |  | duration |
+| transaction.op | Short code identifying the [type of operation](https://develop.sentry.dev/sdk/performance/span-operations/) the span is measuring. | x |  |  | string |
+| transaction.status | Describes the status of the span/transaction. Check out our [Transaction Payloads documentation](https://develop.sentry.dev/sdk/event-payloads/transaction/) for all possible statuses. | x |  |  | string |
+| url | A specific URL that the user visited during the replay. |  |  | x | string |
```

Comment:
```suggestion
| urls | A specific URL that the user visited during the replay. |  |  | x | string |
```
Does url exist for replays? Should be urls. We alias in the search box to url but its not a response field.